shell/browser/api/atom_api_desktop_capturer.h

```diff
@@ -64,6 +52,7 @@ class DesktopCapturer : public mate::TrackableObject<DesktopCapturer>,
   bool capture_window_ = false;
   bool capture_screen_ = false;
   bool fetch_window_icons_ = false;
+  base::WeakPtrFactory<DesktopCapturer> weak_ptr_factory_{this};
```

Comment:
we want `weak_ptr_factory_` is the last member variable in `DesktopCapturer` so that it's
destroyed first (ensures that if any class methods bound to a wk ptr instance
are `Run()` during teardown, they wont actually be executed)